examples/client/client.c

```diff
@@ -366,6 +366,28 @@ static THREAD_RET readPeer(void* in)
 }
 #endif /* !SINGLE_THREADED && !WOLFSSL_NUCLEUS */
 
+#if defined(WOLFSSL_PTHREADS) && defined(WOLFSSL_TEST_GLOBAL_REQ)
+
+static int callbackGlobalReq(WOLFSSH *ssh, void *buf, word32 sz, int reply, void *ctx)
+{
+    char reqStr[] = "SampleRequest";
+
+    if ((WOLFSSH *)ssh != *(WOLFSSH **)ctx)
```

Comment:
Is it required that `ctx` is a pointer to a pointer to the WOLFSSH object, or is this just a test that the assigned `ctx` is really being passed to the callback?

Reply:
Nevermind. Of course it is. It is in a code block named WOLFSSL_TEST_GLOBAL_REQ.